katello/hooks/pre/30-upgrade.rb

```diff
@@ -34,9 +34,11 @@ def migrate_foreman
 end
 
 def postgresql_10_upgrade
+  start_postgresql
+  (name, owner, enconding, collate, ctype, privileges) = %x{runuser postgres -c 'psql -lt | grep -E "^\s+postgres"'}.chomp.split('|').map(&:strip)
```

Comment:
Nit: usually a fan of long options in scripts (`--list --tuples-only` instead of `-lt`)